Jenkinsfile

```diff
@@ -0,0 +1,70 @@
+#!groovy
+
+pipeline {
+  agent none
+
+  options {
+    disableConcurrentBuilds()
+  }
+
+  stages {
+    stage('Build Docker image') {
+      agent any
+      steps {
+        script {
+          def dockerRepoName = 'zooniverse/seven-ten'
+          def dockerImageName = "${dockerRepoName}:${BRANCH_NAME}"
+          def newImage = docker.build(dockerImageName)
```

Comment:
```suggestion
          def newImage = docker.build(dockerImageName)
          newImage.push()
```

Reply:
I suggest moving this so that it pushes all branches to DockerHub (just in case we decide to temporarily deploy a different branch at any point).